requirements.txt

```diff
@@ -13,4 +13,4 @@ aiohttp==0.20.2
 aiohttp-auth==0.1.1
 aiohttp-session[secure]==0.5.0
 aiozmq==0.7.1
-msgpack-python==0.4.7
\ No newline at end of file
+msgpack-python==0.4.7
```

Comment:
it's because I added a blank line at the end of the file. I can't figure out why the github diff shows it this way